datanode/src/storage_api_test.py

```diff
@@ -223,96 +221,96 @@ def testIncorrectPutsOnGridCells(self):
     j = json.loads(result.data)
     s = j['sync_token']
     self.assertEqual(404, self.app.put(
-        '/GridCellMetaDatas/1/1/1',
```

Comment:
It's not important, but isn't the original 4-space indent following a paren the standard rather than the 2-spaces this PR changes it to?